dodola/tests/test_services.py

```diff
@@ -582,48 +582,22 @@ def test_aiqpd_train(tmpdir, monkeypatch):
     )
 
     # remove leap days
-    temp_slice = convert_calendar(ds["air"], target="noleap")
+    ref_fine = convert_calendar(ref_fine["temperature"], target="noleap")
 
     # take the mean across space to represent coarse reference data for AFs
-    temp_slice_mean = temp_slice.mean(["lat", "lon"])
-    # then tile it to be on the same grid as the fine reference data
-    temp_slice_mean_resampled = temp_slice_mean.broadcast_like(temp_slice)
-
-    # need to create some fake bias corrected data so that we can use it to downscale
-    with set_options(sdba_extra_output=True):
-        quantiles = equally_spaced_nodes(620, eps=None)
-        QDM = QuantileDeltaMapping(
-            kind="+",
-            nquantiles=quantiles,
-            group=sdba.Grouper("time.dayofyear", window=31),
-        )
-        QDM.train(temp_slice_mean + 2, temp_slice_mean)
-        fake_biascorrected = QDM.adjust(temp_slice_mean + 4)
-        # this is necessary to make sim_q a coordinate on 'scen'
-        fake_biascorrected = (
-            fake_biascorrected["scen"]
-            .assign_coords(sim_q=fake_biascorrected.sim_q)
-            .to_dataset()
-        )
-    # make bias corrected data on the fine resolution grid
-    biascorrected = fake_biascorrected["scen"].broadcast_like(temp_slice)
+    ds_ref_coarse = ref_fine.mean(["lat", "lon"])
+    ref_coarse = ds_ref_coarse.broadcast_like(ref_fine)
```

Comment:
Minor suggestion - maybe put a quick comment in here explaining why you're using `broadcast_like` here.

Reply:
Done - thanks!